Tests/Engine/Helper.tests.ps1

```diff
@@ -26,4 +26,11 @@ Describe "Test Directed Graph" {
             $digraph.IsConnected('v1', 'v4') | Should -BeTrue
         }
     }
+
+    Context "Runspaces should be disposed" {
+        It "Running analyzer 100 times should not result in additional runspaces" {
+        $null = 1..100 | %{ Invoke-ScriptAnalyzer -ScriptDefinition 'gci' }
+        (Get-Runspace).Count | Should -BeLessThan 10
```

Comment:
Indentation and more info to assertion message
```suggestion
            (Get-Runspace).Count | Should -BeLessThan 10 -Because 'Number of Runspaces should not exceed size of runspace pool cache in CommandInfoCache'
```